### ISSUE TITLE
将bert当作层使用

### ISSUE THREAD

**Issue by BeautyChesnut** [bojone/bert4keras#96]
苏神您好，我想将这里的bert当作一个层来使用，如有两个句子A和B，分别输入到bert中获取输出，如A = bert(A)
B = bert(B)
此处的bert是共用同一个，但是我没有找到相关的接口，请问您的模型中是否有相关的接口可以用，请赐教。谢谢！

**Comment by bojone** [comment]
您好，请认真学习keras后再来用bert4keras。这是keras问题，不是bert4keras问题。

https://keras.io/getting-started/functional-api-guide/

**Comment by BeautyChesnut** [comment]
非常感谢，之前一直被一个小细节迷惑，跟普通的共享层没区别，谢谢苏神～

**Comment by YoungXu06** [comment]
还是不知道怎么在一个模型里用A = bert(A) B = bert(B)这样的操作，比如：
`def build_model(model_path, mode='bert', LR=1e-5, DR=0.2, a_max_len=200, b_max_len=500):
    config_path = model_path+'bert_config.json'
    checkpoint_path = model_path+'bert_model.ckpt'
    
    bert = build_transformer_model(
        config_path = config_path,
        checkpoint_path = checkpoint_path
        )

    input_a_ids = Input(shape=[None,], dtype='int32', name='a_tokens_input')
    segment_a_ids = Input(shape=[None,], dtype='int32', name='a_segments_input')
    
    input_b_ids = Input(shape=[None,], dtype='int32', name='b_tokens_input')
    segment_b_ids = Input(shape=[None,], dtype='int32', name='b_segments_input')
    
    a_bert_output = bert([input_a_ids, segment_a_ids])
    b_bert_output = bert([input_b_ids, segment_b_ids])
    
    a_pool = GlobalAveragePooling1D()(a_bert_output)
    b_pool = GlobalAveragePooling1D()(b_bert_output)
    
    merged = concatenate([a_pool, b_pool])   
    output = Dropout(rate=DR)(merged)
    output = Dense(units=1, activation='softmax')(output)
    model = Model(inputs=[input_a_ids, segment_a_ids, input_b_ids, segment_b_ids], outputs=[output])
    
    model.compile(
        loss='sparse_categorical_crossentropy',
        optimizer=Adam(LR),
        metrics=['accuracy'],
    )
    print(model.summary())
    return model`

会报错的，还是不明白bert如何像普通的层一样使用？

**Comment by YoungXu06** [comment]
https://github.com/bojone/bert4keras/issues/130  遇到了同样的问题，请问如何解决呢？

**Comment by BeautyChesnut** [comment]
> #130 遇到了同样的问题，请问如何解决呢？
我就是像你代码里面那么用的，我用的没有问题
a_bert_output = bert([input_a_ids, segment_a_ids])
b_bert_output = bert([input_b_ids, segment_b_ids])
